diff --git a/node_modules/neode/build/index.js b/node_modules/neode/build/index.js
index a31a57d..da574bd 100644
--- a/node_modules/neode/build/index.js
+++ b/node_modules/neode/build/index.js
@@ -566,8 +566,8 @@ var Neode = /*#__PURE__*/function () {
     }
   }], [{
     key: "fromEnv",
-    value: function fromEnv() {
-      require('dotenv').config();
+    value: function fromEnv(path = '.env') {
+      require('dotenv').config({ path: path });
 
       var connection_string = "".concat(process.env.NEO4J_PROTOCOL, "://").concat(process.env.NEO4J_HOST, ":").concat(process.env.NEO4J_PORT);
       var username = process.env.NEO4J_USERNAME;
